fastparquet/api.py

```diff
@@ -78,6 +80,31 @@ def _parse_header(self, f, verify=True):
     def columns(self):
         return [f.name for f in self.schema if f.num_children is None]
 
+    def sorted_columns(self):
+        """
+        The columns that are known to be sorted
+
+        Returns
+        -------
+        A set of column names
+
+        See Also
+        --------
+        statistics
+        """
+        s = statistics(self)
+        columns = self.columns
+        out = dict()
+        for c in columns:
+            min, max = s['min'][c], s['max'][c]
+            if any(x is None for x in min + max):
+                continue
+            if (sorted(min) == min and
+                sorted(max) == max and
+                all(map(operator.ge, max, min))):
+                out[c] = {'min': min, 'max': max}
```

Comment:
Ah yes, I suppose I'm finding partition-wise sorted columns

At the end of the day this is all the information I need personally.  I'd prefer to avoid mucking with metadata unnecessarily.  I suggest that I move this off to be a separate utility function out of the main class namespace.